components/brave_wallet/browser/solana_block_tracker.cc

```diff
@@ -5,39 +5,60 @@
 
 #include "brave/components/brave_wallet/browser/solana_block_tracker.h"
 
+#include <utility>
+
 #include "base/bind.h"
+#include "base/callback_helpers.h"
 #include "base/logging.h"
 #include "brave/components/brave_wallet/browser/json_rpc_service.h"
 
 namespace brave_wallet {
 
+namespace {
+
+constexpr base::TimeDelta kExpiredTimeDelta =
+    base::Seconds(kBlockTrackerDefaultTimeInSeconds);
```

Comment:
20 seconds is chosen for Ethereum because blocks are ~12-14s but it may make sense to alter this for for Solana

Reply:
Opened a followup at https://github.com/brave/brave-browser/issues/21994, thanks.